azure-pipelines.yml

```diff
@@ -52,11 +52,6 @@ steps:
     key: 'yarn | "$(Agent.OS)" | yarn.lock'
     restoreKeys: 'yarn | "$(Agent.OS)"'
     path: $(YARN_CACHE_FOLDER)
-
-- task: NodeTool@0
-  inputs:
-    versionSource: 'spec'
-    versionSpec: '>=18.x'

```

Comment:
Do we not want to specify the node.js version anymore?

Reply:
Well, the machine as some sort of default that was bumped to v18 in February, they use the latest LTS which I think is a good policy... I put that in place earlier for the workaround as it was apparently sometimes using v16 and sometimes v18 (I guess depending on the machine assigned or something).

I do believe we should support 16/18 as I doubt a lot of developers use v18 locally...

Reply:
And since we no longer have the workaround, we build fine on v16 and v18 if this PR is merged.